packages/core/admin/admin/src/content-manager/components/BlocksInput/BlocksContent.tsx

```diff
@@ -26,6 +29,169 @@ const StyledEditable = styled(Editable)`
+  const { formatMessage } = useIntl();
+
+  const handleMoveBlock = React.useCallback(
+    (newIndex: Array<number>, currentIndex: Array<number>) => {
+      const [newNode] = Editor.node(editor, newIndex);
+      const [draggedNode] = Editor.node(editor, currentIndex);
+
+      Transforms.moveNodes(editor, {
+        at: currentIndex,
+        to: newIndex,
+      });
+
+      // Add 1 to the index for the live text message
+      const currentIndexPosition = [currentIndex[0] + 1, ...currentIndex.slice(1)];
+      const newIndexPosition = [newIndex[0] + 1, ...newIndex.slice(1)];
+
+      setLiveText(
+        formatMessage(
+          {
+            id: getTrad('components.Blocks.dnd.reorder'),
+            defaultMessage: '{item}, moved. New position in the editor: {position}.',
+          },
+          {
+            item: `${name}.${currentIndexPosition.join(',')}`,
+            position: `${newIndexPosition.join(',')} of ${editor.children.length}`,
+          }
+        )
+      );
+
+      // If a node is dragged out of the list block then convert it to a paragraph
+      if (newNode?.type !== 'list-item' && draggedNode?.type === 'list-item') {
+        Transforms.setNodes(editor, { type: 'paragraph' }, { at: newIndex });
+      }
+
+      // If a node is dragged into the list block then convert it to a list-item
+      if (newNode?.type === 'list-item' && draggedNode?.type !== 'list-item') {
+        if (newIndex[0] > currentIndex[0]) {
+          // Node is dragged downwards inside list
+          newIndex[0] -= 1;
+        }
+
+        const { type: _type, children: _children, ...extra } = draggedNode;
+        const attributesToClear: Record<string, null> = {};
+        Object.keys(extra).forEach((key) => {
+          attributesToClear[key] = null;
+        });
+
+        Transforms.setNodes(editor, { ...attributesToClear, type: 'list-item' }, { at: newIndex });
+      }
+    },
+    [editor, formatMessage, name, setLiveText]
+  );
+
+  const [{ handlerId, isDragging, handleKeyDown }, blockRef, dropRef, dragRef] = useDragAndDrop(
+    !disabled && canDrag,
+    {
+      type: `${ItemTypes.BLOCKS}._${name}`,
+      canDropHandler() {
+        return canDrop;
+      },
+      index,
+      item: {
+        displayedValue: children,
+      },
+      onMoveItem: handleMoveBlock,
+    }
+  );
+
+  const composedBoxRefs = composeRefs(blockRef, dropRef);
+
+  return (
+    <Box ref={composedBoxRefs}>
+      {isDragging ? (
+        <Box
+          borderStyle="solid"
+          borderColor="secondary200"
+          borderWidth="2px"
+          width="calc(100% - 24px)"
+          marginLeft="auto"
+          marginTop={2}
+          marginBottom={2}
+        >
+          {/* Rendering the block is necessary for Slate to locate the node, 
+              however while dragging, we hide the rendered block */}
+          <Box display="none">{children}</Box>
```

Comment:
I think you could pass display none to the parent, no need to have 2 boxes

Reply:
we do need to display dragging line

Reply:
ah right sorry